src/MapStyle.ts

```diff
@@ -0,0 +1,14 @@
+export interface MapStyle {
+  title: string,
+  url: string,
+};
+
+export const MapStyles = {
+  Light: { title: 'Light', url: 'mapbox://styles/mapbox/light-v10' } as MapStyle,
+  Dark: { title: 'Dark', url: 'mapbox://styles/mapbox/dark-v10' } as MapStyle,
+  Satellite: { title: 'Satellite', url: 'mapbox://styles/mapbox/satellite-v9' } as MapStyle,
+}
+
+export function getDefaultMapStyleForMode (darkmode: Boolean) {
+  return darkmode ? MapStyles.Dark : MapStyles.Light;
+}
```

Comment:
Just a tip, I wouldn't bother putting this in it's own file, as it's only consumed in one location, I'll convert the generic "dropdown" component to a more aptly named `MapStyleDropdown` or something along those lines, but I'm waiting for a bug to be fixed before that, so this is ok for now.